### PR TITLE
Support tfjs converter options in JAX conversion

### PR DESCRIPTION
**Pull request by mattsoulanille**
Support configuring the options, such as quantization, passed to tfjs converter during conversion of a jax model.

To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7331)

### CODE REVIEW

**Comment by mattsoulanille** [comment]
Fixes #7330, #7260

File: tfjs-converter/python/tensorflowjs/converters/jax_conversion.py [hunk 1]
```diff
@@ -60,7 +60,8 @@ def convert_jax(
     *,
     input_signatures: Sequence[Tuple[Sequence[Union[int, None]], DType]],
     model_dir: str,
-    polymorphic_shapes: Optional[Sequence[Union[str, PolyShape]]] = None):
+    polymorphic_shapes: Optional[Sequence[Union[str, PolyShape]]] = None,
+    **tfjs_converter_params):
```
[the remark this answers is not in the document]
Comment (chunnienc) [reply tfjs-converter/python/tensorflowjs/converters/jax_conversion.py:64]: The former is more pythonic.

**Review by pyu10055 — APPROVED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7331)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @chunnienc and @mattsoulanille)

___
*[`tfjs-converter/python/tensorflowjs/converters/jax_conversion.py` line 64 at r1](https://reviewable.io/reviews/tensorflow/tfjs/7331#-NN92v9QnBsovkWkC9lc-r1-64:-NND-Mjq9SgtULTKyxyO:bjplmcm) ([raw file](https://github.com/tensorflow/tfjs/blob/a042af80a8c50d60e17f3f335ee79ccd36256b7c/tfjs-converter/python/tensorflowjs/converters/jax_conversion.py#L64)):*
<details><summary><i>Previously, chunnienc wrote…</i></summary><blockquote>

The former is more pythonic.
</blockquote></details>

thanks, it will be good to update the doc for the extra params

**Review by chunnienc — APPROVED** [review]

**Review by mattsoulanille — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7331)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @chunnienc and @pyu10055)

___
*[`tfjs-converter/python/tensorflowjs/converters/jax_conversion.py` line 64 at r1](https://reviewable.io/reviews/tensorflow/tfjs/7331#-NN92v9QnBsovkWkC9lc-r1-64:-NND9qJp3Bk2-Q4-q0Np:b-ky38rc) ([raw file](https://github.com/tensorflow/tfjs/blob/a042af80a8c50d60e17f3f335ee79ccd36256b7c/tfjs-converter/python/tensorflowjs/converters/jax_conversion.py#L64)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

thanks, it will be good to update the doc for the extra params
</blockquote></details>

good point. Done.

**Review by pyu10055 — APPROVED** [review]
> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7331)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @chunnienc)

**Review by chunnienc — APPROVED** [review]

**Pull request merged**